opm/autodiff/SolventPropsAdFromDeck.cpp

```diff
@@ -55,7 +55,7 @@ SolventPropsAdFromDeck::SolventPropsAdFromDeck(DeckConstPtr deck,
 
     auto tables = eclState->getTableManager();
     // pvt
-    const auto& pvdsTables = tables->getPvdsTables();
+    const TableContainer& pvdsTables = tables->getPvdsTables();
```

Comment:
Could TableContainer be a template? Then this line could read just as before, and line 68 could avoid the template argument.

The code is ok for merging though if this goes through as-is in opm-parser.